app/styles/app/layouts/billing.scss

```diff
@@ -155,6 +155,108 @@
     margin-bottom: 1em;
     margin-top: 1em;
 
+    &__addons_box {
+      background-color: #fff;
+      border-radius: 5px;
+      padding: 6px 13px;
+      border: 1px solid $pebble-grey;
+      margin-bottom: 10px;
+      width: 208px;
+      height: 114px;
+      flex-basis: auto;
+      margin-right: 5px;
+
+      &--desc {
+        color: $asphalt-grey;
+        font-weight: 600;
+        font-size: 0.8em;
+        margin: 5px 0 12px 0;
+        color: #9d9d9d;
+        text-transform: uppercase;
+
+        span {
+          color: $cement-grey;
+          font-size: .8em;
+          font-style: italic;
+        }
+      }
+
+      &--help {
+        &:hover {
+          border-bottom: 1px solid;
+          cursor: pointer;
+        }
+      }
+
+      &--price {
+        margin: 0 0 10px 0;
+        font-weight: 300;
+        font-size: 1.7em;
+        text-align: left;
+        color: $oxide-blue;
+
+        span {
+          color: $cement-grey;
+          font-size: .5em;
+        }
+      }
+
+      &--name {
+        margin: 0px 0 10px 0;
+        color: #666666;
```

Comment:
This is [asphalt-grey](https://github.com/travis-ci/travis-web/blob/master/app/styles/app/vars.scss#L24) color.